src/audio/codec_adapter/codec/cadence.c

```diff
@@ -94,6 +94,34 @@ static struct cadence_api cadence_api_table[] = {
 #endif
 };
 
+static int cadence_codec_resolve_api(struct comp_dev *dev, uint32_t api_id)
+{
+	struct module_data *codec = comp_get_module_data(dev);
+	uint32_t no_of_api = ARRAY_SIZE(cadence_api_table);
```

Comment:
`unsigned int` should be fine for this. My personal current favourite for this kind of variables would be `n_apis`, not saying it's the absolute best, but `no_of_<single_form_noun>` seems even less clear to me :-)